azext_edge/edge/command_map.py

```diff
@@ -204,17 +204,35 @@ def load_iotops_commands(self, _):
     with self.command_group(
         "iot ops connector opcua trust",
         command_type=connector_resource_ops,
+        is_preview=True,
```

Comment:
the parent "iot ops connector" group is set as preview, so you don't need to explicitly set preview for all the children groups.